terraform/storage/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "aws_account_id" {
+  description = "AWS Account ID"
+  type        = string
+}
+
+variable "aws_access_key_id" {
+  description = "AWS Access Key ID"
+  type        = string
+}
+
+variable "aws_secret_access_key" {
+  description = "AWS Secret Access Key"
+  type        = string
+}
+
+variable "service_subdomain" {
+  description = "Service subdomain"
+  type        = string
+  default     = "tech-audit-tool-api"
+}
+
+variable "domain" {
+  description = "Domain name (e.g., sdp-dev)"
+  type        = string
+  default     = "sdp-dev"
+}
+
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "eu-west-2"
+}
+
+variable "project_tag" {
+  description = "Project"
+  type        = string
+  default     = "TAT"
+}
+
+variable "team_owner_tag" {
+  description = "Team Owner"
+  type        = string
+  default     = "Knowledge Exchange Hub"
+}
+
+variable "business_owner_tag" {
+  description = "Business Owner"
+  type        = string
+  default     = "DST"
+}
+
+variable "container_ver" {
```

Comment:
Are the ECR variables needed here if this terraform is just making a bucket?

I know the vars are used in `main.tf` but if only resourcing storage, why is ECR needed?

Reply:
Removed.